packages/react/menu/src/Menu.tsx

```diff
@@ -41,7 +41,7 @@ const SUB_CLOSE_KEYS: Record<Direction, string[]> = {
 
 const MENU_NAME = 'Menu';
 
-type MenuContextValue = {
+type MenuRootContextValue = {
   isSubmenu: false;
```

Comment:
Would there be any value in updating these to `isRootMenu` instead for consistency?

One advantage I can think of, which I noted a few times whilst reviewing the PRs but never commented on is that in ternaries root would come before sub which to me is more readable or makes more sense to my brain 😀

But again at the time before I didn't think that was enough of a reason to comment on it or ask for a change.
Perhaps now it is?

Reply:
> Would there be any value in updating these to `isRootMenu` instead for consistency?

I don't think so. The intent is different here. In `DropdownMenu` I need to know if it is the root menu to do root menu specific things related to the new trigger part.

In `Menu` we want to know if they are in a submenu.

Reply:
Fair enough, I don't think it matters much what you test for as in the end it's the same result but I can see why we'd want to keep it closer to what we're catering for. Looking at it, it would reverse some ternaries in the way I mentioned but more of them would then be the wrong way, so you're probably right 😀

Reply:
If you would prefer `root` parts to render before sub parts in the ternary then I could always do `!context.isSubmenu`? 😏